pulser-core/pulser/backend/remote.py

```diff
@@ -103,6 +103,14 @@ def get_status(self) -> SubmissionStatus:
         """Gets the status of the remote submission."""
         return self._connection._get_submission_status(self._submission_id)
 
+    def _get_available_result(self, submission_id: str) -> dict[str, Result]:
+        """Return available results and ignore jobs with no results."""
+        return {
+            k: v
+            for k, v in self._connection._query_result(submission_id).items()
```

Comment:
You'll want to select the subset of job IDs withint this `RemoteResults` instance

Reply:
Done